io_burst/burst_io.sh

```diff
@@ -353,9 +354,26 @@ do
 done
 
 cd /tmp
-mv  $run_dir/burst_io_results/$results_file $results_file
+$TOOLS_BIN/test_header_info --front_matter --results_file $results_file --host $to_configuration --sys_type $to_sys_type --tuned $to_tuned_setting --results_version $burst_io_version --test_name $test_name_run
+cat  $run_dir/burst_io_results/$results_file >> $results_file
+results=`grep 0: $results_file`
+test_results="Ran"
+if [ $?0 -ne 0 ]; then
+	test_reults="Failed"
+else
+	for i in $results; do
+		value=`echo $i | cut -d':' -f 2`
+		if [[ $value != ?(-)+([[:digit:]]) ]]; then
```

Comment:
Might want to quote the regex, just in case we ever need to add a wildcard to it